docs/src/snap/howto/backup-restore.md

```diff
@@ -0,0 +1,304 @@
+# Backup and restore
+
+[Velero][] is a popular open source backup solution for Kubernetes.
+Its core implementation is a controller running in the cluster that oversees
+the backup and restore operations. The administrator is given a CLI tool to
+schedule operations and/or perform on-demand backup and restores. This CLI
+tool creates Kubernetes resources that the in-cluster Velero controller acts
+upon. During installation the controller needs to 
+be [configured with a repository (called a 'provider')][providers], where
+the backup files are stored.
+
+This document describes how to setup Velero with the MinIO provider acting
+as an S3 compatible object store.
+
+
+## What you will need
+
+- A running Canonical Kubernetes with DNS enabled
+- MinIO (install described below)
+- Velero (install described below))
+- An example workload
```

Comment:
- Credentials for AWS or another supported cloud provider ?

Reply:
I don't think AWS is required -- just that MinIO + Velero uses credentials that LOOK like AWS creds.  Really its just S3 creds.  I've updated the docs to make that more clear